cafebabel/static/js/edit-article.js

```diff
@@ -142,5 +138,5 @@ Listener.keyup()
 
 window.addEventListener('load', () => {
   const tags = new Tags(document.querySelector('.tags'))
```

Comment:
If I understand correctly the constructor of `Tags` doesn't take any parameter now?